schema/BUS_LAB_NON_CONT_ACC.json

```diff
@@ -305,12 +272,10 @@
     "contribution_date": {
       "title": "CONTRIBUTION DATE",
       "description": "",
-      "type": "integer",
-      "minimum": 0,
-      "maximum": 99999999,
-      "examples": [
-        20120615
-      ],
+      "type": ["string", "null"],
+      "minLength": 10,
+      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$",
```

Comment:
I ran into the issue where the minlength had to be set, but i think ajv was working alright with the format `date-time`. what didn't work with `date`